add_new_host.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+load 'ambari_client.rb'
+require 'yaml'
+require 'byebug'
+
+# Parse command line.
+cluster_name = ARGV[0]
+datanode = ARGV[1]
+
```

Comment:
Fine for this commit, but going forward we should add code to check the command line arguments and print out a help banner if it's not as expected